### DISCUSSION TITLE
Train pifpaf with dataset having different input size

### DISCUSSION THREAD

**Discussion by jagdishbhanushali** [openpifpaf/openpifpaf#477] [Q&A]
Hello,
Very impressive work.
I am working on project where I have to detect skeletons of people who are very far. So for people who are close for them we can have 17 key points, but people who are very far from camera it doesn't make sense to have 17 key points for them and we have annotated them with less number of keypoints.
How can I train and predict on such dataset where I have 17 keypoints for closer people and less than 17 keypoints for people who are far.

Thanks,
Jagdish Bhanushali

**Comment by svenkreiss** [comment]
The annotations don't have to be complete. Also in COCO, most/many person instances are annotated with less than 17 keypoints.

In principle, you can train on two datasets simultaneously. We have done that for our tracking results. That should also work for two image datasets with two different skeletons. You'd get two predictions then and have to do the NMS yourself.
